robin/model/builder.py

```diff
@@ -45,11 +45,24 @@ def load_pretrained_model(model_path, model_base, model_name, load_8bit=False, l
             # use_flash_attention_2 = True,
         )
     
-    token_num, tokem_dim = model.lm_head.out_features, model.lm_head.in_features
-    if model.lm_head.weight.shape[0] != token_num:
-        model.lm_head.weight = torch.nn.Parameter(torch.empty(token_num, tokem_dim, device=model.device, dtype=model.dtype))
-        model.model.embed_tokens.weight = torch.nn.Parameter(torch.empty(token_num, tokem_dim, device=model.device, dtype=model.dtype))
-
+    ## Below two paragraph of code are commented out because we don't actually
```

Comment:
As you mention here, this needs to be fixed. We don't train this with this *right now*, but if we do, the code will break.